tests/modules/nf-core/parabricks/fq2bam/main.nf

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PARABRICKS_FQ2BAM } from '../../../../../modules/nf-core/parabricks/fq2bam/main.nf'
```

Comment:
It would be good to have several aliases for this `PARABRICKS_FQ2BAM` modules, so that we can set up ext.task.args in the paired nextflow,config file.